starknet_py/tests/e2e/docs/guide/test_declaring_contracts.py

```diff
@@ -16,9 +16,9 @@ async def test_declaring_contracts(
     (compiled_contract, class_hash) = map_compiled_contract_and_class_hash_copy_1
 
     # docs: start
-    # Account.sign_declare_v2 and Account.sign_declare_v3 take string containing a compiled contract (sierra)
+    # Account.sign_declare_v3 take string containing a compiled contract (sierra)
     # and a class hash (casm_class_hash)
-    # They return DeclareV2 and DeclareV3 respectively
+    # They return DeclareV3
```

Comment:
```suggestion
    # It returns DeclareV3 struct
```
Or this line can be also removed